vhs-frontend/src/i18n/config.ts

```diff
@@ -0,0 +1,17 @@
+import { initReactI18next } from 'react-i18next';
+import i18n from 'i18next';
+
+import translation from './locales/en/translation.json';
+
+export const resources = {
+  en: {
+    translation,
+  },
+};
+
+i18n.use(initReactI18next).init({
+  debug: true,
+  lng: 'en',
+  keySeparator: '.',
```

Comment:
The dot should be the default key separator, so you should be able to remove this explicit setting.

Reply:
Also, add this export at the end of this file: `export { i18n };`.